app/templates/components/validated-input.hbs

```diff
@@ -0,0 +1,24 @@
+{{!-- Copied from ember-cpi-validations/tests/dummy/app - should eventually be part of ember-osf but getting errors with dependencies --}}
```

Comment:
That said, some of the stuff I: read implied that v-get broke because I was in an addon. If you haven't tried it in this app, that's worth a look before just commenting it out.